sources/packages/web/src/components/layouts/FullPageContainer.vue

```diff
@@ -11,7 +11,7 @@
     <v-container :fluid="fullWidth">
       <template v-if="layoutTemplate === LayoutTemplates.CenteredCard">
         <v-row justify="center">
-          <v-card class="mt-4 p-4 w-100" :class="widthClass">
+          <v-card class="mt-4 p-4 w-100 overflow" :class="widthClass">
```

Comment:
If the solution is suggested in the below comment, we can apply the style only on the page where it is needed.
https://github.com/bcgov/SIMS/pull/3374/files#r1626665008